frappe_graphql/utils/resolver/document_resolver.py

```diff
@@ -48,21 +67,32 @@ def _get_value(fieldname, ignore_translation=False):
 
         # ignore_doc_resolver_translation might be helpful for overriding document_resolver
         # which might be a simple wrapper around this function (document_resolver)
-        if not ignore_translation and isinstance(value, str) and not frappe.flags.ignore_doc_resolver_translation:
+        if not ignore_translation and isinstance(
+                value, str) and not frappe.flags.ignore_doc_resolver_translation:
             return frappe._(value)
```

Comment:
@fahimalizain we do not want to translate [default_fields](https://github.com/frappe/frappe/blob/fec76ba950d759d01ced61d371d3877cfb39a782/frappe/model/__init__.py#L86) particularly [doctype].name or [doctype].doctype

Reply:
I think its not related to this PR. We have to fix it separately. @e-lobo